cmd_watch.go

```diff
@@ -251,6 +251,8 @@ func evalWorkflow(client *circleci.Client, wfID string, jobName string) (anyRunn
 			// it's not configured to run this build (because of a tag or something)
 			anyBlocked = true
 			continue
+		case "not_running":
+			fallthrough
```

Comment:
```suggestion
		case "not_running":
			// not_running is the same as queued
			fallthrough
```
I can't find a better description but maybe a comment is helpful for future us.